compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/destructuring-mixed-scope-and-local-variables-with-default.expect.md

```diff
@@ -56,71 +56,78 @@ function useFragment(_arg1, _arg2) {
 }
 
 function Component(props) {
```

Comment:
this one changes because `useFragment` is defined locally, so we don't pickup the global declaration of `useFragment` that we inject for tests (which comes from shared-runtime)